docs/home.md

```diff
@@ -2,49 +2,76 @@
+
+At Sunrise Stake we believe in the power of Regenerative Finance to create positive outcomes for people and the planet.
+So we’ve created a non-custodial and permissionless protocol to send the yield earned through staking SOL towards retiring carbon tokens and other climate-positive projects.
+Sunrise Stake is one of the first ReFi projects on the Solana blockchain; we provide a seamless way for holders of SOL to participate in the ReFi movement, while simultaneously strengthening the Solana blockchain.
 
-By staking SOL with Sunrise, you are strengthening the Solana blockchain,
-while simultaneously offsetting your climate impact.
 
 ### What is ReFi?
 
-Regenerative Finance, or ReFi, is the union of web3 decentralised finance
-with regenerative economics. 
+Regenerative Finance, or ReFi, is the union of web3 decentralised finance with regenerative economics.
+
+Regenerative economics is a theoretical framework for understanding and guiding economic activity that is focused on creating positive social and ecological outcomes.
+
+This approach to economics emphasizes the importance of building and maintaining healthy ecosystems and communities, and views economic activity as a means to that end rather than an end in itself.
+
+### What are the benefits of staking with Sunrise Stake?
+
+#### Supporting carbon offset projects comes at no cost
+Users deposit their SOL with Sunrise Stake and receive gSOL (green SOL) in return as a wrapped token representing the total amount staked. 
+gSOL is a synthetic derivative of SOL, and can be used in the same way as SOL, by protocols or recipients that support it.
+For example, if you are asked to pay a recipient in SOL, you can pay them in gSOL instead. The recipient can then choose to either keep the gSOL, or unstake it and receive SOL.
+The gSOL to SOL ratio is always 1:1 thus stakers experience no fund loss in the process. Additionally, gSOL is a fully fungible token - stakers can continue to trade it, leaving their SOL to earn rewards for nature-based projects!
+
+#### Transparent unstaking 
+
+The deposited stake retains full liquidity and can be traded back to SOL by  the user any time. At present, no fees (beyond the Solana network fee of 5000 lamports) apply to unstaking high values when the user withdraws with a delay of 2-3 days, otherwise a fee of 0.3% is incurred for immediate withdrawals. This is done to secure sufficient liquidity of the staking pool. 
+
+#### Liquid Staking
+Liquid staking is a method of staking that allows stakers to unstake without the typical 2-3 day withdrawal period.
+Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) and [SolBlaze] (https://stake-docs.solblaze.org/) protocols, which allows liquid staking through the use of liquidity pools.
+
+_NOTE: In the future, Sunrise Stake aims at offering entirely feeless liquid unstaking. Sunrise will also proxy additional staking protocols._
```

Comment:
Synthesise this section with the above. The Note is no longer fully applicable, now that we have (at least up to a limit) feeless withdrawals.

Reply:
Done